helpers/fail2ban

```diff
@@ -100,6 +100,15 @@ ignoreregex =
     ynh_add_config --template="f2b_jail.conf" --destination="/etc/fail2ban/jail.d/$app.conf"
     ynh_add_config --template="f2b_filter.conf" --destination="/etc/fail2ban/filter.d/$app.conf"
 
+    # if "$logpath" doesn't exist (as if using --use_template argument), assign
+    # "$logpath" using the one in the package's conf/f2b_jail.conf template
+    if [ -z "$logpath" ]; then
+        # the first sed deleteds possibles spaces and the second one extract the path
+        logpath=$(grep logpath "$YNH_APP_BASEDIR/conf/f2b_jail.conf" | sed "s/ //g" | sed "s/logpath=//g")
+        # replace the '__APP__' by the real app name in the path
+        logpath=$(sed "s/__APP__/$app/g")
```

Comment:
uh 😔
what do we do so? :<

Reply:
maybe we could instead check that `$logpath` is defined (i mean in the 'if' later), with eg `if [ -n "${logpath:-}"] && [[ -f "$logpath" ]]` 

and simply ignore the case where logpath is not defined, because those 3 apps should probably not use the "external" template and use the args instead just so we have a single consistent way to define fail2ban stuff accross all apps

Reply:
yes, but that would make two different behaviors:
when using `--use_template`, the log file would not be created if needed, whereas when using `--logpath=log_file --failregex=filter`, it would be created.